src/Pages/Adopt.jsx

```diff
@@ -289,6 +330,12 @@ export const Adopt = () => {
           </Typography>
           <Box sx={{ width: "600px", maxWidth: "100%" }}>
             <form onSubmit={handleSubmit(onSubmit)}>
+              <Typography
+                variant="h3"
+                sx={{ mb: 2 }}
+              >
+                General Info
```

Comment:
Great job adding these headers! It is a very long form, and I think these headers split things up nicely.